zipkin-server/src/main/java/zipkin2/server/internal/mysql/ZipkinMySQLStorageProperties.java

```diff
@@ -112,7 +112,7 @@ private String determineJdbcUrl() {
     }
 
     StringBuilder url = new StringBuilder();
-    url.append("jdbc:mysql://");
+    url.append("jdbc:mariadb://");
```

Comment:
mariadb recently enforces the jdbc url name has to be branded as such. That said, our images still install mysql so we test based on that.